calculator.js

```diff
@@ -1,2 +1,38 @@
-var calculator = {
+var value=0;
+var calculator= {
+add: function (num1,num2){
+      console.log(num1+num2);
+      value =(num1 + num2);
+      return "The sum of these two number is : "+ value;
```

Comment:
You don't want to return the string. You want to return the numerical value (i.e., `return num1 + num2`).